core/src/main/scala/org/apache/spark/metrics/sink/Slf4jSink.scala

```diff
@@ -25,6 +25,9 @@ import com.codahale.metrics.{MetricRegistry, Slf4jReporter}
 import org.apache.spark.SecurityManager
 import org.apache.spark.metrics.MetricsSystem
 
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
```

Comment:
The problem you describe is that log messages aren't plumbed through to a logger, right? that shouldn't require changing appender config for log4j. Your JIRA says "Slf4jsink.scala should have class name for log4j to print metrics in log files" but your code has a package name.